File: AtCoder_Beginner_Contest_130/B.py
# -*- Coding: utf-8 -*-
N, X = map(int, input().split())
L = list(map(int,input().split()))

ans = 1
p = 0
p += L[ans]
while p <= X:
    p += L[ans]
    ans += 1

print(ans) 

# N,X = map(int,input().split())
# L = list(map(int,input().split()))

# ans = 1 #位置0も1にカウントするから
# p = 0 #今のポジション
# for i in range(N):
#     p += L[i]
#     if p <= X:
#          ans += 1
#     else:
#         break

# print(ans)        

File: AtCoder_Beginner_Contest_130/B.cpp
#include <iostream>
using namespace std;

int main(){
    int n,x;
    cin >> n >> x;
    vector<int> l(n)
    for (int i=0;i < n;i++)
        cin >> l[i]
    }
    int ans = 1;
    int p = 0;
    for (int i=0;i < x;i++){
        p += l[i]
        if (p <= x) ans++;
    }
    cout << ans << endl;
    return 0
}
